src/codegen_call_io.cpp

```
@@ -77,84 +77,43 @@ llvm::Value *CodeGen::emitBuiltinRegex(const CallExpr &e) {
-    // read_text(path) -> Result<str, Error>
-    if (e.callee == "read_text") {
-        llvm::Value *ptr = emitIOCall(e.callee, 1, ptrTy_);
-        return wrapPtrAsResult(ptr);
-    }
-
-    // bytes_to_str(bytes) -> Result<str, Error>
-    if (e.callee == "bytes_to_str") {
-        llvm::Value *ptr = emitIOCall(e.callee, 1, ptrTy_);
-        return wrapPtrAsResult(ptr);
-    }
+static const char *IO_ERR = "__ry_get_last_error";
```

Clang (14.0.6)

[warning] variable 'IO_ERR' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)